src/Features/CSharp/Portable/MakeAnonymousFunctionStatic/MakeAnonymousFunctionStaticCodeFixProvider.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file should go into the shared project here: https://github.com/dotnet/roslyn/tree/master/src/Analyzers/Core/CodeFixes

Reply:
This comment still holds